scylladb/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/gofiber/storage/scylladb
 
 go 1.20
 
+replace github.com/gocql/gocql => github.com/scylladb/gocql v1.11.1
```

Comment:
I mean the import on line 8 should be the one in the replace. I'm not sure if dependabot will pick it up as a "replace" directive

Reply:
From the tests it should be fine.
https://github.com/dependabot/dependabot-core/blob/main/go_modules/spec/fixtures/go_mods/go.mod